Python/phate/phate.py

```diff
@@ -443,98 +520,102 @@ def set_params(self, **params):
+            self.decay = params["a"]
             reset_kernel = True
-            del params['a']
-        if 'knn' in params and params['knn'] != self.knn:
-            self.knn = params['knn']
+            del params["a"]
+        if "knn" in params and params["knn"] != self.knn:
+            self.knn = params["knn"]
             reset_kernel = True
-            del params['knn']
-        if 'decay' in params and params['decay'] != self.decay:
-            self.decay = params['decay']
+            del params["knn"]
+        if "decay" in params and params["decay"] != self.decay:
+            self.decay = params["decay"]
             reset_kernel = True
-            del params['decay']
-        if 'n_pca' in params:
-            if self.X is not None and params['n_pca'] >= np.min(self.X.shape):
-                params['n_pca'] = None
-            if params['n_pca'] != self.n_pca:
-                self.n_pca = params['n_pca']
+            del params["decay"]
+        if "n_pca" in params:
+            if self.X is not None and params["n_pca"] >= np.min(self.X.shape):
+                params["n_pca"] = None
+            if params["n_pca"] != self.n_pca:
+                self.n_pca = params["n_pca"]
                 reset_kernel = True
-                del params['n_pca']
-        if 'knn_dist' in params and params['knn_dist'] != self.knn_dist:
-            self.knn_dist = params['knn_dist']
+                del params["n_pca"]
+        if "knn_dist" in params and params["knn_dist"] != self.knn_dist:
+            self.knn_dist = params["knn_dist"]
             reset_kernel = True
-            del params['knn_dist']
-        if 'n_landmark' in params and params['n_landmark'] != self.n_landmark:
-            if self.n_landmark is None or params['n_landmark'] is None:
+            del params["knn_dist"]
+        if "n_landmark" in params and params["n_landmark"] != self.n_landmark:
+            if self.n_landmark is None or params["n_landmark"] is None:
                 # need a different type of graph, reset entirely
                 self._reset_graph()
             else:
-                self._set_graph_params(n_landmark=params['n_landmark'])
-            self.n_landmark = params['n_landmark']
-            del params['n_landmark']
+                self._set_graph_params(n_landmark=params["n_landmark"])
+            self.n_landmark = params["n_landmark"]
+            del params["n_landmark"]
 
         # parameters that don't change the embedding
-        if 'n_jobs' in params:
-            self.n_jobs = params['n_jobs']
-            self._set_graph_params(n_jobs=params['n_jobs'])
-            del params['n_jobs']
-        if 'random_state' in params:
-            self.random_state = params['random_state']
-            self._set_graph_params(random_state=params['random_state'])
-            del params['random_state']
-        if 'verbose' in params:
-            self.verbose = params['verbose']
+        if "n_jobs" in params:
```

Comment:
Aww, I liked single ticks

Reply:
`black` says otherwise, fight them not me

Reply:
Fight `black` not me